netscaler-adc/docs/NetScalerADC.md

```diff
@@ -0,0 +1,93 @@
+</td></tr><tr><td>Destination IP</td><td>Specify the IP address or range of IP addresses to match against the destination IP address of an incoming IPv4 packet
+</td></tr><tr><td>Other Fields</td><td>Specify fields in the JSON format to be sent as json_data according to NetScaler REST API Documentation. https://developer-docs.netscaler.com/en-us/adc-nitro-api/current-release/configuration/ns/nsacl.html
+</td></tr></tbody></table>
+#### Output
+
+ The output contains a non-dictionary value.
+### operation: Get NetScaler ACL Resource
+#### Input parameters
+<table border=1><thead><tr><th>Parameter</th><th>Description</th></tr></thead><tbody><tr><td>ACL Name</td><td>Specify the Name for the extended ACL rule that you want to retrieve. By default all ACL will be fetched
+</td></tr><tr><td>Page Size</td><td>Specify the Number of results to return per page.
+</td></tr><tr><td>Page Number</td><td>Specify the page number of results to be fetched
+</td></tr><tr><td>Other Fields</td><td>Specify fields in the JSON format to be sent as json_data according to NetScaler REST API Documentation. https://developer-docs.netscaler.com/en-us/adc-nitro-api/current-release/configuration/ns/nsacl.html
+</td></tr></tbody></table>
+#### Output
+
+ The output contains a non-dictionary value.
+### operation: Delete NetScaler ACL Resource
+#### Input parameters
+<table border=1><thead><tr><th>Parameter</th><th>Description</th></tr></thead><tbody><tr><td>ACL Name</td><td>Specify the Name for the extended ACL rule that you want to delete
+</td></tr></tbody></table>
+#### Output
+
+ The output contains a non-dictionary value.
+### operation: Delete NetScaler ACL Resource
+#### Input parameters
+<table border=1><thead><tr><th>Parameter</th><th>Description</th></tr></thead><tbody><tr><td>ACL Name</td><td>Specify the Name for the extended ACL rule that you want to delete
+</td></tr></tbody></table>
+#### Output
+
+ The output contains a non-dictionary value.
+### operation: Change NetScaler ACL Resource State
+#### Input parameters
+<table border=1><thead><tr><th>Parameter</th><th>Description</th></tr></thead><tbody><tr><td>ACL Name</td><td>Specify the Name for the extended ACL rule that you want to delete
+</td></tr><tr><td>Action</td><td>Specify the Action for the extended ACL rule to apply
+</td></tr></tbody></table>
+#### Output
+
+ The output contains a non-dictionary value.
+## Included playbooks
+The `Sample - netscaler-adc - 1.0.0` playbook collection comes bundled with the NetScaler ADC connector. These playbooks contain steps using which you can perform all supported actions. You can see bundled playbooks in the **Automation** > **Playbooks** section in FortiSOAR&trade; after importing the NetScaler ADC connector.
+
+- Change NetScaler ACL Resource State
+- Create NetScaler ACL Resource
+- Delete NetScaler ACL Resource
+- Delete NetScaler ACL Resource
```

Comment:
This is extra one : - Delete NetScaler ACL Resource
Add delete in last as CRUD sequence